Ryujinx.HLE/HOS/Services/Nifm/StaticService/IGeneralService.cs

```diff
@@ -71,6 +73,27 @@ public ResultCode GetCurrentIpAddress(ServiceCtx context)
             return ResultCode.Success;
         }
 
+        [Command(18)]
+        // GetInternetConnectionStatus() -> nn::nifm::detail::sf::InternetConnectionStatus
+        public ResultCode GetInternetConnectionStatus(ServiceCtx context)
+        {
+            if (!NetworkInterface.GetIsNetworkAvailable())
+            {
+                return ResultCode.NoInternetConnection;
+            }
+
+            InternetConnectionStatus result = new InternetConnectionStatus
```

Comment:
```suggestion
            InternetConnectionStatus internetConnectionStatus = new InternetConnectionStatus
```
Maybe? Or at least just `status`?